AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/AudioSelectionAdapter.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.samples.communication.calling.helpers;
+
+import android.content.Context;
+import android.widget.PopupWindow;
+import androidx.core.content.ContextCompat;
+import com.azure.samples.communication.calling.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AudioSelectionAdapter extends BottomCellAdapter {
+
+    public AudioSelectionAdapter(final PopupWindow popupWindow,
```

Comment:
Instead of providing a direct reference to the `PopupWindow`, we can provide a `onSelect` callback.

Reply:
Addressed.